<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="RHJ-14F-D547" name="com.scripts.netSuite.approvalWorkflow.TS_NS_6694_Approval_Workflow_Market_under_10k_creditRebill" tests="1" failures="0" timestamp="18 Feb 2020 12:48:43 GMT" time="38.889" errors="1">
  <testcase name="TS_NS_6694_Approval_Workflow_Market_under_10k_creditRebill" time="38.889" classname="com.scripts.netSuite.approvalWorkflow.TS_NS_6694_Approval_Workflow_Market_under_10k_creditRebill">
    <error type="junit.framework.AssertionFailedError" message="Adjustment / Special Billing  is not found">
      <![CDATA[junit.framework.AssertionFailedError: Adjustment / Special Billing  is not found
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at com.framework.Generic.Verify(Generic.java:295)
at com.framework.Generic.clickButton(Generic.java:333)
at com.netsuite.common.NS_Billing_AdjustmentAndSpecialBilling.selectAdjSplBillingInBilling(NS_Billing_AdjustmentAndSpecialBilling.java:170)
at com.scripts.netSuite.approvalWorkflow.TS_NS_6694_Approval_Workflow_Market_under_10k_creditRebill.TS_NS_6694_Approval_Workflow_Market_under_10k_creditRebill(TS_NS_6694_Approval_Workflow_Market_under_10k_creditRebill.java:95)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- TS_NS_6694_Approval_Workflow_Market_under_10k_creditRebill -->
</testsuite> <!-- com.scripts.netSuite.approvalWorkflow.TS_NS_6694_Approval_Workflow_Market_under_10k_creditRebill -->
